firmware/controllers/math/pid_auto_tune.cpp

```diff
@@ -86,7 +86,7 @@ void PID_AutoTune::SetLookbackSec(int value)
 
 double inline PID_AutoTune::fastArcTan(double x)
 {
-  // source: �Efficient approximations for the arctangent function�, Rajan, S. Sichun Wang Inkol, R. Joyal, A., May 2006
+  // source: �Efficient approximations for the arctangent function�, Rajan, S. Sichun Wang Inkol, R. Joyal, A., May 2006
```

Comment:
this does not looks right at least on the web

Looks like I will be closing this PR, let's please have a separate commit with ONLY const?

Reply:
Agh! This source we imported from elsewhere uses a bunch of non-ascii characters, like those quotes.